sale_stock_picking_blocking/tests/test_sale_stock_picking_blocking.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo.exceptions import ValidationError
+from odoo.tests import common
+
+
+class TestSaleDeliveryBlock(common.TransactionCase):
+    def setUp(self):
```

Comment:
Could you change this to classmethod setUpClass ? It will allow to not load this at each test.